src/transforms/lua/v2/mod.rs

```diff
@@ -329,7 +334,7 @@ impl RuntimeTransform for Lua {
 }
 
 #[cfg(test)]
-fn format_error(error: &rlua::Error) -> String {
+pub(crate) fn format_error(error: &rlua::Error) -> String {
```

Comment:
Why this change? Doesn't seem like we're using it anywhere new.

Reply:
I... have no idea.